internal/configuration/env_config.go

```diff
@@ -57,6 +58,7 @@ func init() {
 	settings["GetDebugMode"] = Setting{"JC_DEBUG_MODE", defaultDebugMode, []func(interface{}, string) error{util.IsBool}}
 	settings["GetHTTPSEnabled"] = Setting{"JC_ENABLE_HTTPS", defaultHTTPSEnabled, []func(interface{}, string) error{util.IsBool}}
 	settings["GetGatewayTimeout"] = Setting{"JC_GATEWAY_TIMEOUT", defaultGatewayTimeout, []func(interface{}, string) error{util.IsDuration}}
+	settings["GetAllowedOrigins"] = Setting{"JC_ALLOWED_ORIGINS", defaultAllowedOrigins, []func(interface{}, string) error{util.IsNotEmpty}}
```

Comment:
defaultAllowedOrigins should be `[]string` instead so that the caller need not worry about the implementation detail (how the string is delimited)

Reply:
Not sure, how this can be done. I don't see much value here anyway. I would rather skip this. If you want this feel free to modify this PR.